.github/workflows/test.yml

```diff
@@ -9,6 +9,11 @@ on:
 jobs:
   test:
     runs-on: ubuntu-latest
+    services:
+      remote-chrome:
+        image: browserless/chrome:1.57-puppeteer-19.2.2
```

Comment:
maybe it would make sense to also use a version matrix here, but I have no experience with that..